src/core/congestion_control.h

```diff
@@ -171,6 +171,32 @@ typedef struct QUIC_CONGESTION_CONTROL {
 
 } QUIC_CONGESTION_CONTROL;
 
+
+//
+// V1 supports careful resume on 1 path per remote endpoint
+//
+typedef struct QUIC_CONN_CAREFUL_RESUME_V1 {
```

Comment:
If this is the struct that you will use to put into the ticket, then I have a few suggestions:
- You shouldn't use enums, only well-known types like uint32_t
- You shouldn't need 64-bits for time, depending on the units, which could probably be in milliseconds. 4 billion milliseconds is over 46 days, which is much longer than it takes light to go across the solar system
- I think the congestion control algorithm should be first, because the rest of the types could be different per CC
- You probably want to pack the struct